### ISSUE TITLE
[Bug]:   Visual artifacts when using DPM++ schedulers and SDXL without the refiner model

### ISSUE THREAD

**Issue by meditat2001** [openvinotoolkit/stable-diffusion-webui#109]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

When using OpenVino + SDXL 1.0 trained Model with 1024x1024 px and dpm samplers are used, artifacts occur (no refiner used).

Machine: Intel Iris Xe shared memory, 16GB Ram, PCIe 4, Windows 11, latest Chrome + Opera, OpenVino Toolkit, latest Intel Graphic driver.

### Steps to reproduce the problem

Known Bug, please look at: 
https://github.com/huggingface/diffusers/issues/5433

maybe solution is:   https://github.com/huggingface/diffusers/pull/5541

Interestingly  https://github.com/vladmandic/automatic with use of openvino 
has not the problem. 

### What should have happened?

Generate no artifacts

### Sysinfo
{
    "Platform": "Windows-10-10.0.22631-SP0",
    "Python": "3.10.6",
    "Version": "1.6.0",
    "Commit": "e5a634da06c62d72dbdc764b16c65ef3408aa588",
    "Script path": "D:\\sd6\\sd",
    "Data path": "D:\\sd6\\sd",
    "Extensions dir": "D:\\sd6\\sd\\extensions",
    "Checksum": "564ad7b401cbfad4ee980db3c4aff31d7caa8904547bcd1593cff5be45671b22",
    "Commandline": [
        "launch.py",
        "--skip-torch-cuda-test",
        "--precision",
        "full",
        "--no-half"
    ],
    "Torch env info": {
        "torch_version": "2.1.0+cpu",
        "is_debug_build": "False",
        "cuda_compiled_version": null,
        "gcc_version": null,
        "clang_version": null,
        "cmake_version": null,
        "os": "Microsoft Windows 11 Pro",
        "libc_version": "N/A",
        "python_version": "3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)] (64-bit runtime)",
        "python_platform": "Windows-10-10.0.22631-SP0",
        "is_cuda_available": "False",
        "cuda_runtime_version": null,
        "cuda_module_loading": "N/A",
        "nvidia_driver_version": null,
        "nvidia_gpu_models": null,
        "cudnn_version": null,
        "pip_version": "pip3",
        "pip_packages": [
            "numpy==1.23.5",
            "open-clip-torch==2.20.0",
            "pytorch-lightning==1.9.4",
            "torch==2.1.0",
            "torchdiffeq==0.2.3",
            "torchmetrics==1.4.0.post0",
            "torchsde==0.2.5",
            "torchvision==0.16.0"
        ],
        "conda_packages": null,
        "hip_compiled_version": "N/A",
        "hip_runtime_version": "N/A",
        "miopen_runtime_version": "N/A",
        "caching_allocator_config": "",
        "is_xnnpack_available": "True",
        "cpu_info": [
            "Architecture=9",
            "CurrentClockSpeed=907",
            "DeviceID=CPU0",
            "Family=205",
            "L2CacheSize=5120",
            "L2CacheSpeed=",
            "Manufacturer=GenuineIntel",
            "MaxClockSpeed=2419",
            "Name=11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz",
            "ProcessorType=3",
            "Revision="
        ]
    },
    "Exceptions": [],
    "CPU": {
        "model": "Intel64 Family 6 Model 140 Stepping 1, GenuineIntel",
        "count logical": 8,
        "count physical": 4
    },
    "RAM": {
        "total": "16GB",
        "used": "10GB",
        "free": "5GB"
    },
    "Extensions": [
        {
            "name": "adetailer",
            "path": "D:\\sd6\\sd\\extensions\\adetailer",
            "version": "a89c01d3",
            "branch": "main",
            "remote": "https://github.com/Bing-su/adetailer"
        },
        {
            "name": "deforum-for-automatic1111-webui",
            "path": "D:\\sd6\\sd\\extensions\\deforum-for-automatic1111-webui",
            "version": "5d63a339",
            "branch": "automatic1111-webui",
            "remote": "https://github.com/deforum-art/deforum-for-automatic1111-webui.git"
        },
        {
            "name": "sd-webui-agent-scheduler",
            "path": "D:\\sd6\\sd\\extensions\\sd-webui-agent-scheduler",
            "version": "721a36f5",
            "branch": "main",
            "remote": "https://github.com/ArtVentureX/sd-webui-agent-scheduler"
        }
    ],
    "Inactive extensions": [],
    "Environment": {
        "COMMANDLINE_ARGS": "--skip-torch-cuda-test --precision full --no-half",
        "GRADIO_ANALYTICS_ENABLED": "False"
    },
---

### What browsers do you use to access the UI ?

Google Chrome, Other

### Console logs

```Shell
---
```
